### DISCUSSION TITLE
Update Free Policy does not work from RTL for Eclair 0.6.2

### DISCUSSION THREAD

**Discussion by xbukorg** [ACINQ/eclair#2019] [Developers]
Hi Guys,

The Update Free Policy not work from RTL 0.11.2-beta if Eclair 0.6.2-6817d6f is installed. 

![image](https://user-images.githubusercontent.com/60103142/137747397-42da10a5-575b-461e-afae-571b3be026dd.png)

![image](https://user-images.githubusercontent.com/60103142/137747182-93331990-ef13-41bd-babc-e89fe8466a2d.png)

It work for Eclair  0.6.1.
Regards

**Comment by thomash-acinq** [comment]
Relay fees are now set per node instead of per channel, see [release notes](https://github.com/ACINQ/eclair/blob/master/docs/release-notes/eclair-v0.6.2.md#per-node-relay-fees).

**Reply by xbukorg** [reply]
Can you shift this bug to RTL responsible person? 
Maybe it should be fixed on RTL side.

**Reply by t-bast** [reply]
It should indeed fixed on the RTL side.
Please open a bug to them as a user of RTL.
